### ISSUE TITLE
Mouse pointer disappears when no other windows are open

### ISSUE THREAD

**Issue by zh4ngx** [raphamorim/rio#647]
On NixOS + Wayland -
If I open rio with no other windows open, the mouse disappears. This happens whether or not I'm typing.

**Comment by ozmodeuz** [comment]
also getting this behaviour with nixos + wayland; happy to help debug

**Comment by zh4ngx** [comment]
> also getting this behaviour with nixos + wayland; happy to help debug

Does the problem go away as soon as you open another window?

**Comment by ozmodeuz** [comment]
yeah, if Rio is the first window my mouse cursor disappears when over the terminal, once a second app is open it behaves as normal. not tried it with 2 Rio instances yet, will do later

**Comment by zh4ngx** [comment]
I tried closing all other windows after opening a bunch (in my case, multiple chrome instances), and the mouse cursor disappears again
